scss/mixins/_grid-framework.scss

```diff
@@ -22,7 +20,10 @@
     @include media-breakpoint-up($breakpoint) {
       // Work around cross-media @extend (https://github.com/sass/sass/issues/1050)
       %grid-column-float-#{$breakpoint} {
+        $gutter: map-get($gutters, $breakpoint);
         float: left;
+        padding-left:  ($gutter / 2);
+        padding-right: ($gutter / 2);
```

Comment:
This breaks the padding for flex grid. Consider renaming to `%grid-column-#{$breakpoint}` and moving the `$enable-flex` check inside.